backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/domain/discord/DiscordCheckVerificationRequest.java

```diff
@@ -18,15 +19,27 @@ public class DiscordCheckVerificationRequest {
     private String eventId;
 
     @NotBlank
-    private String stakeAddress;
+    private String walletId;
+
+    @Builder.Default
+    private Optional<WalletType> walletIdType = Optional.of(WalletType.CARDANO);
```

Comment:
I suggest to call variable: walletType than walletIdType.

Reply:
done:)